webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -42,6 +36,7 @@ export enum WorkspaceDetailsTab {
 
 const WorkspaceDetails: React.FC = () => {
   const rbacEnabled = useFeature().isOn('rbac');
+  const mockWorkspaceMembers = useFeature().isOn('mock_workspace_members');
```

Comment:
What does this mock stand of?

Reply:
If someone is using `f_mock_workspace_members=on` to show users in the Workspace Members tab / `pages/WorkspaceDetails/WorkspaceMembers.tsx`, then I don't want the API response to overwrite the members list.